app/views/spree/admin/invoices/_invoices_table.html.haml

```diff
@@ -7,12 +7,12 @@
       %th= t(:status)
       %th= t(:file)
   %tbody
-    - @order.invoices.each do |invoice|
+    - @order.invoices.order('number desc').each do |invoice|
       - tr_class = cycle('odd', 'even')
       - tr_id = spree_dom_id(invoice)
       %tr{:class => tr_class, "data-hook" => "invoice_row", :id => tr_id}
         %td.align-center.created_at
-          = pretty_time(invoice.date)
+          = invoice.presenter.invoice_date
```

Comment:
Sorry, I try to retrieve some information, but couldn't find if this handle i18n?

Reply:
As far as I can see we are going in circle here, `Invoice::DataPresenter` delegates `date` to `Invoice` so I think we should revert this change. Plus the change doesn't handle i18n

Reply:
@jibees  you're right. I fixed that by  adding a new helper.

@rioug 
I don't see a circle
```ruby
class Invoice
  class DataPresenter
     delegate :data, :date, to: :invoice
....
    def invoice_date
      date
    end
  end
end
```
Instead, we can replace that code with 
```ruby
    delegate :date, to: :invoice, prefix: true
```

Reply:
The presenter delegate `date` to `invoice`, right ? so calling `invoice.presenter.invoice_date` should return the same thing as `invoice.date` . So I don't see the point of calling the presenter here.